lib/capybara/mechanize/node.rb

```diff
@@ -2,6 +2,7 @@
 
 class Capybara::Mechanize::Node < Capybara::RackTest::Node
   def click(keys = [], **options)
+    options.delete(:offset)
```

Comment:
I decided to just delete the option, since it is known and it really does not make a difference in this case.
This is the same as `Capybara::RackTest::Node` does.

This way the user does not need change the configuration on update.

The alternative fix would have been to disable the new default during the test suite, but that does not sound like the best approach.